lib/sky-pages-module-generator.js

```diff
@@ -14,6 +16,33 @@ const routeGenerator = require('./sky-pages-route-generator');
  * @returns {string} source
  */
 function getSource(skyAppConfig) {
+  const cssFilePaths = skyAppConfig.skyux.app &&
+    skyAppConfig.skyux.app.styles &&
+    skyAppConfig.skyux.app.styles;
```

Comment:
Is this a typo or missing `.length`?  

I've recently started using https://www.npmjs.com/package/lodash.get to handle this exact scenario in a single line.

const get = require('lodash.get');
const cssFilesPaths = get(skyAppConfig, 'skyux.app.styles', []);

Reply:
You can thank @blackbaud-brandonhare for that tip.